Assets/_BForBoss/_Core/Scripts/Environment/PlayerTriggerAreaEffect.cs

```diff
@@ -0,0 +1,142 @@
+using System;
+using Perigon.Character;
+using Perigon.Entities;
+using Perigon.Utility;
+using Sirenix.OdinInspector;
+using UnityEngine;
+
+namespace BForBoss
+{
+    [RequireComponent(typeof(Collider))]
+    public class PlayerTriggerAreaEffect : MonoBehaviour
+    {
+        private enum HealthEffect
+        {
+            Damage,
+            Heal
+        }
+
+        private enum SpeedModifier
+        {
+            SlowDown,
+            SpeedUp
+        }
+        
+        
+        [Serializable]
+        private class PlayerMutation
```

Comment:
Can this be placed in a separate file, And also don't call it `PlayerMutation` 
Just call it `Mutation`. 
PlayerMutation implies it's only for players, as opposed to anything else.

Reply:
I think `Mutation` is to vague as it can refer to anything that mutates, maybe too be more specific `PlayerTriggerAreaMutation`??